### ISSUE TITLE
WowUp detect when same addon has been installed or could be installed from 2 different providers

### ISSUE THREAD

**Issue by Raina1007** [WowUp/WowUp#1365]
**Is your feature request related to a problem? Please describe.**
B-Morgan would like it when an addon may be accidentally be installed from two providers or 1 version is installed from another provider and user is going to install it a 2nd time w/o realizing to get a popup about it being installed already. To avoid the solution of having to uninstall 1 then do a rescan folders to remove the 2nd one and then reinstall. As rescan folders can cause other unintended consequences such as put other addons on ignore. 

**Describe the solution you'd like**
A popup or another method to show it is already installed from 1 provider when going to install from another provider

**Describe alternatives you've considered**
Unknown

**Additional context**
N/A

**Comment by MrUzagi** [comment]
Unless I've missed some crucial information this looks similar to what I sometimes come across when switching provider (Curse -> Wago for example) and the addon isn't recognized as having multiple providers. I usually just install the new one, thereby having two addons in the WoWUp list (but installed to the same folder(s)), uninstall the one I don't want and finally re-install the one left. Doing that leaves just the one add-on without the need for a rescan.

TL;DR
* Uninstall all addons except the wanted one.
* Re-install the last one, no rescan needed.

**Comment by jliddev** [comment]
Problem is knowing addon from provider A is the same as provider B. in some cases we have a map of ids we could look thru, but many times we will not.
